src/module/entities/helper.js

```diff
@@ -1046,7 +1087,7 @@ module.exports = class UserProjectsHelper {
 	 * @returns {Promise<Object>} Promise that resolves with fetched documents or rejects with an error.
 	 */
 
-	static entityListBasedOnEntityType(type, pageNo, pageSize, paginate) {
+	static entityListBasedOnEntityType(type, pageNo, pageSize, paginate, language) {
```

Comment:
@MallanagoudaB update function signature,and make sure function won't break if language is not passed